src/main/scala/org/ergoplatform/modifiers/history/BlockTransactions.scala

```diff
@@ -98,28 +117,49 @@ object BlockTransactions extends ApiCodecs {
     for {
       headerId <- c.downField("headerId").as[ModifierId]
       transactions <- c.downField("transactions").as[List[ErgoTransaction]]
+      blockVersion <- c.downField("blockVersion").as[Version]
       size <- c.downField("size").as[Int]
-    } yield BlockTransactions(headerId, transactions, Some(size))
+    } yield BlockTransactions(headerId, blockVersion, transactions, Some(size))
   }
 }
 
 object BlockTransactionsSerializer extends ScorexSerializer[BlockTransactions] {
-
-  override def serialize(obj: BlockTransactions, w: Writer): Unit = {
-    w.putBytes(idToBytes(obj.headerId))
-    w.putUInt(obj.txs.size)
-    obj.txs.foreach { tx =>
+  // See a comment in the parse() function
+  val MagicNumber = 1000000
```

Comment:
Let's call it like `MaxTransactionsInV1Block`

Reply:
done